fbw-common/src/systems/instruments/src/EFB/Localization/data/en.json

```diff
@@ -336,6 +336,76 @@
     },
     "SYNC": "SYNC",
+    "Takeoff": {
+      "AntiIce": "Anti-Ice",
+      "Airport": "Airport",
+      "Calculate": "Calculate",
+      "CentreOfGravity": "TOCG",
+      "Clear": "Clear",
+      "CoGPosition": "CG Position",
+      "CoGPositions": {
+        "Forward": "Forward (<27%)",
+        "Standard": "Standard"
+      },
+      "Configuration": "Configuration",
+      "CrosswindAboveLimitTitle": "Maximum Crosswind Is {max_crosswind} Knots",
+      "CrosswindAboveLimitMessage": "Do you want to perform the computation with {actual_crosswind} knots?",
+      "EnterManually": "Enter Manually",
+      "EntryAngle": "Entry Angle",
+      "EntryAngles": {
+        "0": "0°",
+        "90": "90°",
+        "180": "180°"
+      },
+      "Messages": {
+        "FlexNotPossible": "Flex Not Possible",
+        "InvalidData": "Invalid Inputs",
+        "OperatingEmptyWeight": "Below Operating Empty Weight ({oew} {weight_unit})",
+        "StructuralMtow": "Above Structural MTOW ({structural_mtow} {weight_unit})",
+        "MaximumPressureAlt": "Above Maximum Pressure Altitude ({max_zp} ft)",
+        "MaximumTemperature": "Above Maximum OAT ({tmax}°C)",
+        "CgOutOfLimits": "CG Outside Takeoff Envelope",
+        "VmcgVmcaLimits": "VMCG/VMCA Limits Not Satisfied",
+        "MaximumTireSpeed": "Maximum Tire Speed Not Satisfied",
+        "MaximumTailwind": "Above Maximum Tailwind ({max_tailwind} knots)",
+        "MaximumRunwaySlope": "Beyond Runway Slope Limits (±2%)",
+        "TooHeavy": "Beyond Takeoff Weight Limit - Max {mtow} {weight_unit}",
+        "TooLight": "Below Minimum Takeoff Weight",
+        "HeadwindLimited": "Headwind has been reduced to {max_headwind} knots"
+      },
+      "Packs": "Packs",
+      "Qnh": "QNH",
+      "Runway": "Runway",
+      "RunwayBearing": "Runway Bearing",
+      "RunwayBearingUnit": "°",
+      "RunwayCondition": "Runway Condition",
+      "RunwayConditions": {
+        "Dry": "Dry",
+        "Wet": "Wet",
+        "Contaminated6mmWater": "Water - 6.3 mm/¼\"",
+        "Contaminated13mmWater": "Water - 12.7 mm/½\"",
+        "Contaminated6mmSlush": "Slush - 6.3 mm/¼\"",
+        "Contaminated13mmSlush": "Slush - 12.7 mm/½\"",
+        "ContaminatedCompactedSnow": "Compacted Snow",
+        "Contaminated5mmWetSnow": "Wet Snow - 5 mm/⅕\"",
+        "Contaminated15mmWetSnow": "Wet Snow - 15 mm/⅗\"",
+        "Contaminated30mmWetSnow": "Wet Snow - 30 mm/⅗\"",
+        "Contaminated10mmDrySnow": "Dry Snow - 10 mm/⅖\"",
+        "Contaminated100mmDrySnow": "Dry Snow - 100 mm/4\""
+      },
+      "RunwayElevation": "Runway Elevation",
+      "RunwayElevationUnit": "ft",
+      "RunwayLengthUnitFt": "ft",
+      "RunwayLengthUnitMeter": "m",
+      "RunwaySlope": "Runway Slope",
+      "Temperature": "Temperature",
+      "Thrust": "Thrust",
+      "Title": "Takeoff",
+      "Tora": "TORA",
+      "Weight": "Takeoff Weight",
+      "Wind": "Wind",
+      "WindMagnitudeUnit": "kts or °/kts"
+    },
```

Comment:
note for maintainers: check that localazy strings have been added before merging

Reply:
I've posted the keys in the #localisation channel now. 👍